utility/flight_stow.py

```diff
@@ -32,26 +55,36 @@ def point_ants(ants, dry_run=False):
 	except:
```

Comment:
This would be better as `except Exception:`.  Otherwise it could catch things like "CTRL+C".
(This is also a pep8 guideline)